topics/assembly/tutorials/debruijn-graph-assembly/data-library.yaml

```diff
@@ -1,8 +1,35 @@
 ---
-libraries:
-  - name: assembly_data
-    files:
-      - url: https://zenodo.org/record/582600/files/mutant_R1.fastq
-        file_type: "fastqsanger"
-      - url: https://zenodo.org/record/582600/files/mutant_R2.fastq
-        file_type: "fastqsanger"
+destination:
+  description: Galaxy Training Network Material
+  name: GTN - Material
+  synopsis: Galaxy Training Network Material. See https://training.galaxyproject.org
+  type: library
+items:
+  - description: DNA sequence data has become an indispensable tool for Molecular
+      Biology & Evolutionary Biology. Study in these fields now require a genome sequence
+      to work from. We call this a 'Reference Sequence.' We need to build a reference
+      for each species. We do this by Genome Assembly. De novo Genome Assembly is
+      the process of reconstructing the original DNA sequence from the fragment reads
+      alone.
+    items:
+      - ext: fastq
```

Comment:
fastqsanger?

Reply:
I took the extension given by the Zenodo API. I do not know if we should do this way or if we remove the `ext` info and Galaxy could sniff the correction extension